src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs

```diff
@@ -236,37 +236,40 @@ public int StartDebuggingPE()
                     s_readOnlyDocumentTracker = new VsReadOnlyDocumentTracker(_threadingContext, _encService, _editorAdaptersFactoryService);
                 }
 
-                string outputPath = _project.IntermediateOutputFilePath;
+                _mvid = Guid.Empty;
 
-                // The project doesn't produce a debuggable binary or we can't read it.
-                // Continue on since the debugger ignores HResults and we need to handle subsequent calls.
-                if (outputPath != null)
+                if (_project.CompilationOutputs is CompilationOutputFiles outputFiles)
                 {
+                    // The project doesn't produce a debuggable binary, we can't read it or it's not stored in a file on disk.
```

Comment:
Requirement for storing the file on disk is temporary. It will be removed in the new EnC manager implementation.